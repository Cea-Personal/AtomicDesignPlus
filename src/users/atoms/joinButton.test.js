import React from "react";
import {render,cleanup} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import JoinButton from "./joinButton";
afterEach(cleanup)

it('displays a button',()=>{
    const {getByText, asFragment} = render(<JoinButton/>)
    expect(getByText('Join Now')).toBeDefined()
    expect(asFragment()).toMatchSnapshot()
})