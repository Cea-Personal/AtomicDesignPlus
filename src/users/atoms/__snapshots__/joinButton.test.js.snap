// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays a button 1`] = `
<DocumentFragment>
  <button
    class="sc-bdVaJa bgFwOS"
  >
    Join Now
  </button>
</DocumentFragment>
`;
